srcpkgs/scrcpy/template

```diff
@@ -1,21 +1,20 @@
 # Template file for 'scrcpy'
 pkgname=scrcpy
-version=1.3
+version=1.5
 revision=1
 build_style=meson
-configure_args="-Db_lto=true
```

Comment:
configure_args= is multiline, you need to keep it for the -Dprebuild_server option